src/components/video/OutgoingOngoingCall.tsx

```diff
@@ -50,6 +53,7 @@ const OutgoingOngoingCall = ({ blockedLoading, onEndCall }: OutgoingOngoingCallT
           iconWidth="23px"
           onClick={() => {
             // TODO
+            setToggleVideo(!togglevideo);
```

Comment:
2 things here:
- Can you remove the TODO comment from here
- instead of `setToggleVideo(!togglevideo)` use `setToggleVideo(currToggleVideo => !currToggleVideo)`. Just a React optimization.